github.js

```diff
@@ -327,7 +327,7 @@ GithubLocation.prototype = {
           versions[version] = versionObj;
         }
 
-        resolve({ versions: versions });
+        resolve({ versions: versions, latest: 'master', edge: 'dev' });
```

Comment:
I see you have the checks in the main code, which is great, deleted the comment.